rumqttc/src/v5/mod.rs

```diff
@@ -50,8 +50,9 @@ pub enum Request {
 }
 
 #[cfg(feature = "websocket")]
-type RequestModifierFn =
-    dyn Fn(http::Request<()>) -> Pin<Box<dyn Future<Output = http::Request<()>>>>;
+type RequestModifierFn = (dyn Fn(http::Request<()>) -> Pin<Box<dyn Future<Output = http::Request<()>> + Send>>
+     + Send
```

Comment:
```suggestion
```

Reply:
Didn't get which line exactly is this, but if removed, it won't work. I updated it as per the `Arc<_>` changes, plz have a look once and lmk if this is fixed!